working/0698 - Enhanced Default Constructors/proposal.md

```diff
@@ -13,13 +13,14 @@ Creating a class with a simple constructor is fairly short because of Dart's ini
 
 ```dart
 class Complex {
-  final double real, imaginary;
+	final double real, imaginary;
```

Comment:
Nit: Tab -> spaces.

Reply:
Fixed.

(Pro-tip: When transferring markdown between two computers trough a gmail draft mail, do remember to paste the *source*, not the formatted text).